src/features/newTask.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { defaultEqualityCheck } from "reselect";
```

Comment:
- I suggest only importing the needed functions. Consider removing the `defaultEqualityCheck` since it's not being used anywhere. 
- Consider also refactoring the file by removing unnecessary empty lines.